DEVELOPING.md

```diff
@@ -13,9 +13,19 @@ Install the following extensions:
 
 ## Development workflow
 
+### Visual Studio Code
+
 1. Start the build task (Cmd+Shift+B), it will run TypeScript compiler in backround, watching and recompiling files as you change them. Compilation errors will be shown in the VSCode's "PROBLEMS" window.
 
 
 2. Execute "Test and lint" task (Cmd+Shift+T) to re-run the test suite and lint the code for both programming and style errors. Linting errors will be shown in VSCode's "PROBLEMS" window. Failed tests are printed to terminal output only.
 
-3. Run "npm test" explicitly before committing your changes. This will execute the same sequence as our CI server does.
+3. Run `npm test` explicitly before committing your changes. This will execute the same sequence as our CI server does.
+
+### Other editors/IDEs
+
+1. Start the build process in background via `npm run tsc-watch`. It will run TypeScript compiler in backround, watching and recompiling files as you change them.
```

Comment:
Should be `npm run build:watch`. It's not running as a background task for me (not that I mind it. Like it in the foreground so I can see errors).

Reply:
Good catch. I'll rephrase the sentence to make this more clear.